py3status/modules/process_status.py

```diff
@@ -54,13 +60,14 @@ def _get_text(self):
     def process_status(self):
         if self.process is None:
             color = self.py3.COLOR_BAD
-            text = ERR_NO_PROCESS
+            text = self.py3.safe_format(self.format_error,
```

Comment:
self.format_error does not have any placeholder so I'm not sure why you're passing the state key here